src/org/folio/rest_v2/Eureka.groovy

```diff
@@ -0,0 +1,82 @@
+package org.folio.rest_v2
+
+import org.folio.models.OkapiTenant
```

Comment:
@OHaimanov , @eldiiar-duishenaliev 

Should we use **OkapiTenant** class in the future for Eureka deployment code?

Are we going to switch to **[EurekaTenant](https://github.com/folio-org/pipelines-shared-library/blob/6d954f3680b444f32312eff709e4cfd8cc10b509/src/org/folio/models/EurekaTenant.groovy)** class from Pull Request #567 ?